develop-docs/self-hosted/experimental/monitoring.mdx

```diff
@@ -0,0 +1,184 @@
+---
+title: Self-Hosted Monitoring
+sidebar_title: Monitoring
+sidebar_order: 10
+---
+
+<Alert title="Important" level="warning">
+    These are community-contributed docs. Sentry does not officially provide support for self-hosted configurations beyond the default install.
+</Alert>
+
+This page is considered experimental because everyone will have different setup and requirements for their monitoring system. It is also best to use your existing monitoring system, and try to integrate Sentry's with it, instead of spinning up a new one.
```

Comment:
```suggestion
This page is considered experimental because everyone will have different setup and requirements for their monitoring system. It is also best to use your existing monitoring system, and try to integrate Sentry with it, instead of spinning up a new one.
```